alerts/hbase/README.md

```diff
@@ -0,0 +1,48 @@
+# Alert Policies for Apache HBase in the Ops Agent
+
+## Slow Operations
+
+Slow operations can be an indication that the HDFS may be in an unhealthy state.
+
+## HBase Server Stopped
+
+If the HBase has unexpected downtime, this alert will help teams discern that the server is stopped and unable to function in their environment.
```

Comment:
Can we share how long does it need to be down before the alert policy is triggered?